### PR TITLE
[WIP] Import Cloud Storage backend

### PR DESCRIPTION
**Pull request by leplatrem**
* [x] Imported code from Cliquet 1 code base
* [ ] Setup tests
* [ ] Rebase on master after merge of refactors
* [ ] Polish documentation

### CODE REVIEW

File: kinto/storage.py [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+
+API_PREFIX = "/v0"
+
+
+FILTERS = {
+    COMPARISON.LT: 'lt_',
+    COMPARISON.MIN: 'min_',
+    COMPARISON.MAX: 'max_',
+    COMPARISON.NOT: 'not_',
+    COMPARISON.EQ: '',
```
Comment (tarekziade) [inline kinto/storage.py]: missing string or is this normal ?

Comment (ametaireau) [inline kinto/storage.py]: This is normal: when no filter is specified it means that it's ``field=test``. See [the kinto documentation](http://kinto.readthedocs.org/en/latest/api/cliquet/resource.html#filtering) about filtering.

Comment (ametaireau) [inline kinto/storage.py]: Consider this string as what's prefixed to the name of the field.

**Comment by leplatrem** [comment]
Even though this is pretty cool and fun, we have no use-case so far for a Kinto storage backend.

Dropping.

**Pull request closed without merging**